modules/nuxeo/replace-rendition/designer/custom-dropzone.html

```diff
@@ -0,0 +1,549 @@
+<dom-module id="custom-dropzone" assetpath="custom-dropzone/">
```

Comment:
Could we get a top-level comment on how this differs from the dropzone element already available? It appears to restrict based on `Asset`.  Thanks!